.github/workflows/msal-node-e2e.yml

```diff
@@ -107,6 +108,12 @@ jobs:
         AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
         AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
         AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
+        OBO_WEB_APP_CLIENT_ID: ${{ secrets.OBO_WEB_APP_CLIENT_ID }}
+        OBO_WEB_APP_CLIENT_SECRET: ${{ secrets.OBO_WEB_APP_CLIENT_SECRET }}
+        OBO_WEB_API_CLIENT_ID: ${{ secrets.OBO_WEB_API_CLIENT_ID }}
+        OBO_WEB_API_CLIENT_SECRET: ${{ secrets.OBO_WEB_API_CLIENT_SECRET }}
+        OBO_WEB_API_URL: ${{ secrets.OBO_WEB_API_URL }}
```

Comment:
nit: can we call this APP ID URI or just scope?

Reply:
OBO_WEB_APP_ID_URI or APP_ID_URI?

Reply:
I've realized yesterday that it's the full scope (app id uri + .default), so something like `WEB_API_SCOPE`  would be best for clarity.